.claude/skills/har/scripts/capture_har.py

```
@@ -0,0 +1,148 @@
+                )
+            finally:
+                probe.close()
+            context = browser.new_context(
+                user_agent=user_agent,
+                record_har_path=str(args.output),
+                record_har_content="embed",
+            )
+            page = context.new_page()
+            try:
+                page.goto(args.url, wait_until="networkidle", timeout=args.timeout * 1000)
+            except Exception as exc:  # the partial capture is the interesting one
```

Ruff (0.16.3)

[warning] Do not catch blind exception: `Exception`

(BLE001)